packages/web/README.md

```diff
@@ -2,8 +2,10 @@
 
 ## Como executar o website da plataforma?
 
-1. Inicialize o servidor: `npm run dev`.
-2. O serviço estará disponível em: `http://localhost:8000`.
+1. Crie um arquivo `.env`, baseando-se no arquivo `.env.example`;
+2. Inicialize a api;
+3. Inicialize o servidor: `npm run dev`.
```

Comment:
Acredito que seja melhor usar logo o `npm run start` na raíz do projeto.

Reply:
Eu acho que o melhor aqui é deixar o  `npm run dev` mesmo, porque, geralmente, o usuário, vai querer rodar localmente pra ir testando o código e vendo as mudanças. Quando você roda um `npm run start` ele gera o build de produção e depois roda esse build. Então, mesmo que o usuário modifique o código, ele não vai ter um `fast reloading`. 

Mas a gente pode colocar em uma outra seção (quando André for atualizar a documentação) informando como subir localmente uma versão "de produção" :+1:

Reply:
Esse start é da raiz do projeto, Luiz. Roda todos os três packages em modo de dev :+1:

Reply:
Mas esse comentário aqui não tá no arquivo `packages/web/README.md` não? Ou não estou vendo direito ou é do pacote web...

Reply:
Eu adicionei essa instrução no README do pacote web porque se eu inicializar o servidor web antes da api ele dá erro.

Reply:
`npm run start` da raiz roda em modo dev.

`npm run start:ci` é que roda em modo produção.

Reply:
No final das contas o que eu faço? @nicholasio, @mateus4k , @lcnogueira

Reply:
Pessoal, eu "acho" que tá havendo uma confusão. Alexandre tá atualizando o README do pacote web, que fala dos comandos do pacote web. Não é isso não, @alexandreadames ?

Reply:
Exato @lcnogueira

Reply:
Como eu nunca tinha trabalhando no front eu fui ler o README do web e quando dei o comando `npm run dev` ele deu erro, quando fui ver foi apenas a API que tinha que subir antes.

Reply:
Deixa do jeito que está mesmo 👍

Reply:
Eu inclu apenas dois passos, que julguei importante:
1. Crie um arquivo `.env`, baseando-se no arquivo `.env.example`;
2. Inicialize a api;

Reply:
Desculpa, não tinha visto que tinha mais de um readme. Erro meu 😅😅